src/cryptoadvance/specter/templates/includes/fee-selection.html

```diff
@@ -238,11 +238,20 @@ <h4 slot="title">{{ _("Subtract fees from amount") }}</h4>
         // Gets executed after fee initialisation
         initWithFees() {
             // this.feesSlider.min is set above via min_fee
-            this.feesSlider.max = Math.floor(this.fees["fastestFee"] * 1.4)
-            let sliderMin = this.fees["minimumFee"]
-            let sliderMax = this.fees["fastestFee"]
-            let average = Math.floor((sliderMin + sliderMax) / 2)
-            this.feesSlider.value = average
+            // For low fee environment
+            if (this.fees["fastestFee"] == 1) {
+                this.feesSlider.max = 1.5
+            }
+            else {
+                this.feesSlider.max = Math.floor(this.fees["fastestFee"] * 1.4)
```

Comment:
Assume `this.fees["fastestFee"] == 1.1`, then `floor(1.1*1.4) = 1` -->  Stuck slider. I think `ceil` with minimum 2 is the better option.

Reply:
I'll add that case to the tests and fix it, thanks.

Reply:
Implemented in https://github.com/cryptoadvance/specter-desktop/pull/1878/commits/1f2228fea88826ed799e937857d6a76bb6bacb55